doc/installation.rst

```diff
@@ -3,23 +3,70 @@
 Installation
 ============
 
-splot supports python `3.5`_ and `3.6`_ only. Please make sure that you are
-operating in a python 3 environment.
+Installing dependencies
+-----------------------
 
-Installing released version
----------------------------
+`splot` is compatible with Python `3.6`_ and `3.7`_ and
+depends on GeoPandas 0.4.0 or later and matplotlib 2.2.2 or later.
+Please make sure that you are operating in a Python 3 environment.
 
-giddy is available on the TBA. Therefore, you can either
-install directly with `pip` from the command line::
+splot also uses
 
-  pip install splot
+* numpy
+* seaborn
+* mapclassify
+* Ipywidgets
 
+Depending on your spatial analysis workflow and the PySAL objects
+you would like to visualize, splot relies on:
 
-or download the source distribution (.tar.gz) and decompress it to your selected
-destination. Open a command shell and navigate to the decompressed folder.
-Type::
+PySAL 2.0
+
+or separate packages found in the PySAL stack:
+
+* esda
+* libpysal
+* spreg
+* giddy
+
+
+Installing the newest release
+-----------------------------
+
+There are two ways of accessing splot. First, splot is installed with
+the PySAL 2.0 metapackage through:
+
+```$ pip install -U pysal```
+
+or 
+
+```$ conda install -c conda-forge pysal```
+
+Second, splot can be installed as a separate package. If you are
+using Anaconda, install splot via the conda utility:
+
+```$ conda install -c conda-forge splot```
+
+Otherwise you can install splot from PyPI with pip:
+
+```$ pip install splot```
+
+
+Trouble shooting
+----------------
+Most common installation errors are due to splot's dependency on GeoPandas.
+
+It often helps to first install GeoPandas separately from conda-forge with:
+
+```$ conda install geopandas```
```

Comment:
```
$ conda config --set channel_priority strict
$ conda install --channel conda-forge splot
```

Reply:
this one is about installing geo pandas, not splot, so I'll take up the recommendation to set the channel to conda-forge instead.

Reply:
ah, yes, i must have copy-pasted wrong. The was meant to be `geopandas`.